### ISSUE TITLE
UTF-8 Encoding

### ISSUE THREAD

**Issue by jag3773** [greekcntr/BHP#7]
@greekcntr It looks like the BHP_Data.txt file is encoded as UTF-16, which means that Github and the git utility itself see it as a binary file, even though it is a text file.  This makes the revision history useless (e.g. no way to see what lines were changed in https://github.com/greekcntr/BHP/commit/eb2a59b474ad550556e87ffb602242e70f1cf3c6 ).

Since I don't think there is a particular reason to have this as UTF-16 can we convert it to UTF-8 so that we can make better use of the revision history?  I can submit a pull request with the change if that would be helpful, or you can run the conversion on a unix or linux system with `iconv -f utf-16 -t utf-8 < BHP_Data.txt >BHP_Data.utf8.txt`.  Then overwrite the original BHP_Data.txt file with the UTF-8 version.

However, you probably need to change your database export routine anyway, so maybe that would be the best approach if you want to do this.

**Comment by greekcntr** [comment]
Yeah, that was probably caused because I didn’t remember what I did the previous time. I have a couple of punctuation changes that someone spotted, so I will probably make those changes and put out a new update. But I won’t be able to get to it for about a week as I am leaving for vacation in a few hours. Thanks for letting me know.
